cmake/modules/SIMDExt.cmake

```diff
@@ -1,66 +1,60 @@
 # detect SIMD extentions
 #
-# ARM_NEON_FLAGS
-#
 # HAVE_ARMV8_CRC
```

Comment:
thanks it would be a big change to split this commit and ensure that it builds on each commit. I added verbiage to the commit message around testing. hopefully this satisfies your concern.